snews_pt/core/logging.py

```diff
@@ -0,0 +1,56 @@
+"""
+Modified version of Geoffrey Letner's python 201 logger.
+
+Ref: https://python-tutorial.dev/201/tutorial/logging.html
+"""
+import os
+import time
+from datetime import date
+from socket import gethostname
+from logging import (
+    getLogger,
+    NullHandler,
+    Formatter,
+    FileHandler,
+    DEBUG,
+    INFO,
+    WARNING,
+    ERROR,
+    CRITICAL,
+)
+
+HOST = gethostname()
+
+log_date = date.today().strftime("%Y-%m-%d")
+log_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../../logs")
```

Comment:
We create a directory relative to package. This works for the backend as you have to install the backend manually anyways. However, for the frontend, user likely to install the package via `pip install snews-pt` and it'll store the scripts somewhere in `.local/site-packages/...` whatever. Thus, the logs will also be created somewhere there which is not easy for user to find. 
My understanding is that this style logging is mostly for debugging and we should not expect that from the user

Reply:
That makes sense.  I wasn't thinking about that.